content/docs/rkvst-basics/getting-access-tokens-using-app-registrations/index.md

```diff
@@ -54,12 +54,12 @@ When enabling non-interactive access to RKVST, you ***must*** create your first
 4.  You will then be presented with the `CLIENT_ID` and `SECRET` required by the archivist token endpoint.
 
 {{< caution >}}
-**Caution:** You **must** take note of the `SECRET` at this point - it can **not** be viewed again later.
+**Caution:** You **must** take note of the `SECRET` at this point - it can **not** be viewed again later and you will have to generate a new one.
```

Comment:
Can we state you would need a new one in the event of losing this one?

Reply:
Yep, added!